packages/ppx_tools/ppx_tools.6.1+dune+dev/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Tools for authors of ppx rewriters and other syntactic tools"
+maintainer: "alain.frisch@lexifi.com"
+authors: "Alain Frisch <alain.frisch@lexifi.com>"
+license: "MIT"
+tags: [ "syntax" ]
+homepage: "https://github.com/dune-universe/ppx_tools"
+bug-reports: "https://github.com/ocaml-ppx/ppx_tools/issues"
+dev-repo: "git://github.com/dune-universe/ppx_tools.git"
+build: ["dune" "build" "-p" name "-j" jobs
+    "@runtest" {with-test}]
+depends: [
+  "ocaml" {>= "4.08.0" & < "4.12"}
+  "dune" {>= "1.6"}
+]
+url {
+  src:
+    "https://github.com/dune-universe/ppx_tools/releases/download/v6.1%2Bdune%2Bdev/ppx_tools-6.1.dune.dev.tbz"
```

Comment:
Wouldn't the right version be `6.1+dev+dune` given it is `6.1+dev` plus a dune port?

Reply:
yep you're right! Would that also make the linter happy?

Reply:
Let me fix the whole thing from the start! I'd better have the tag and github releases concur on the version number!

Reply:
It did fix it!